lib/genome/groupers/interaction_grouper.rb

```diff
@@ -0,0 +1,38 @@
+module Genome
+  module Groupers
+    class InteractionGrouper
+      def self.run
+        ActiveRecord::Base.transcation do
+          puts 'reset interactions'
+          reset
+          puts 'add members'
+          add_members
+        end
+      end
+
+      def self.reset
+        DataModel::Interaction.destroy_all
+      end
+
+      def self.add_members
+        DataModel::InteractionClaim.all.each do |interaction_claim|
+          next unless interaction_claim.interaction.nil?
+          drug = interaction_claim.drug_claim.drug
+          next if drug.nil?
+          gene = interaction_claim.gene_claim.gene
+          next if gene.nil?
+          existing_interactions = DataModel::Interaction.where(gene: gene, drug: drug, interaction_type: interaction_claim.interaction_type)
```

Comment:
This doesn't seem to work correctly yet. I get an error: `ERROR:  column interactions.gene does not exist`. The column is called `gene_id` but I thought this would get resolved correctly if the relationships are set. I'm not sure what's wrong with the relationships I defined in the model, though. Probably something stupid.